nailgun/entities.py

```diff
@@ -1970,27 +1983,14 @@ def create_missing(self):
         super(LifecycleEnvironment, self).create_missing()
         if (self.name != 'Library' and  # pylint:disable=no-member
                 not hasattr(self, 'prior')):
-            response = client.get(
-                self.path('base'),
-                data={
-                    u'name': u'Library',
-                    # pylint:disable=no-member
-                    u'organization_id': self.organization.id,
-                },
-                **self._server_config.get_client_kwargs()
-            )
-            response.raise_for_status()
-            results = response.json()['results']
+            results = self.search({'organization'}, {u'name': u'Library'})
```

Comment:
> I think about search functionality always in terms ```.search(file='something')```, so it is not so obvious for me what ```.search(file)``` should do

That's a fine thing to do if you want to write imperative code. But I'm doing my darndest to take advantage of the fact that Python is an object oriented language.